dash_renderer/__init__.py

```diff
@@ -12,8 +12,8 @@
 from .version import __version__
 __file__
 
-_DEFAULT_REACT_VERSION = '16.2.0'
-_REACT_VERSION_TYPES = {'15.4.2', '16.2.0'}
+_DEFAULT_REACT_VERSION = '16.6.3'
```

Comment:
16.7.0 was actually released two days ago, if the goal was the most recent version